extensions/amp-story/1.0/amp-story.js

```diff
@@ -2561,6 +2563,19 @@ export class AmpStory extends AMP.BaseElement {
     );
   }
 
+  /**
+   * Handles the selectPage viewer event.
+   * @param {!JsonObject} data
+   * @private
+   */
+  onSelectPage_(data) {
+    if (data && data['next']) {
```

Comment:
We expose some analytics for publishers to know what triggered the page advancement.
Could you please add a few lines of code to update our analytics service and let it know that this page advancement comes from the viewer?

It should be very straightforward and look like this:

```
        this.storeService_.dispatch(
          Action.SET_ADVANCEMENT_MODE,
          AdvancementMode.VIEWER_SELECT_PAGE
        );
```
Where `AdvancementMode.VIEWER_SELECT_PAGE` has to be defined here:
https://github.com/ampproject/amphtml/blob/6278ede4bf7eba4e4ef4d7fef79351d689dac2d6/extensions/amp-story/1.0/story-analytics.js#L44-L51